x/fosite_storer.go

```diff
@@ -42,5 +42,9 @@ type FositeStorer interface {
 
 	FlushInactiveAccessTokens(ctx context.Context, notAfter time.Time) error
 
+	// flush the consent/login requests from the database.
+	// this will address the database long-term growth issues.
```

Comment:
```suggestion
	// Flush the consent/login requests from the database.
	// This will address the database long-term growth issues discussed in https://github.com/ory/hydra/issues/1574.
```